src/components/FilmCard/FilmCard.jsx

```diff
@@ -0,0 +1,87 @@
+import { Link } from 'react-router-dom';
+import style from './FilmCard.module.css';
+const FilmCard = ( { name, image, genre, id } ) =>
+{
+  const arr = genre.map( ( genre, id ) => id !== genre.length - 1 ? <span className={style.genre} key={id}>{genresConvert( genre )}, </span> : <span className={style.genre} key={id}>{genresConvert( genre )}.</span> )
+  return (
+    <Link to={`/${id}`} className={style.filmCard}>
+      <h3>{name}</h3>
+      <img src={`https://image.tmdb.org/t/p/w500/${image}`} alt="film poster" />
+      <div>Жанр: {arr}</div>
+    </Link>
+  );
+};
+
+export default FilmCard;
+
+// Convert api code genres to russian definition
+function genresConvert( genre )
```

Comment:
Firstly you can create just object with key and value. Secondly you can just `return value` and not to do `genreContent = value; break` everywhere. And move this function to `./helpers` file in the component folder.

Reply:
> Firstly you can create just object with key and value. Secondly you can just `return value` and not to do `genreContent = value; break` everywhere. And move this function to `./helpers` file in the component folder.

Good idea, thank you